### PR TITLE
Fix LCARS menu and improve `header.php` 

### PR DESCRIPTION
**Pull request by rdwebdesign**
**Please make sure you**
 1. Base your code and PRs against the repositories developmental branch.
 2. [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits as we enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions

---
- **What does this PR aim to accomplish?:**

After #2266, the new menu design wasn't working as expected in LCARS theme. The menu HTML and CSS theme needed a few adjustments.

Also, there was a lot of code in file `header.php`.
This code was moved to `sidebar.php` and to new functions in `func.php`.

- **What documentation changes (if any) are needed to support this PR?:**

none

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

File: style/themes/lcars.css [hunk 1]
```diff
@@ -752,10 +752,15 @@ p.login-box-msg,
   padding: 0.5em;
   text-align: left;
   color: #000;
-  background: #CCE6FF;
+  background: #cce6ff;
   margin: 4px 0 0;
 }
 
+/* Hide menu headers if not logged in */
```
Comment (yubiuser) [inline style/themes/lcars.css:760]: Do you really want to hide all the menu headers incl. `Main` and `Support`?

File: scripts/pi-hole/php/header.php [hunk 2]
```diff
@@ -6,156 +6,74 @@
...
-    }
-    else
-    {
-        // Nothing can be colder than -273.15 degree Celsius (= 0 Kelvin)
-        // This is the minimum temperature possible (AKA absolute zero)
-        $celsius = -273.16;
-        // Set templimit to null if no tempsensor was found
-        $temperaturelimit = null;
-    }
+list($celsius, $temperaturelimit) = getTemperature();
```
Comment (yubiuser) [inline scripts/pi-hole/php/header.php:104]:
```suggestion
// getTemperature() defined in `header_func.php`
list($celsius, $temperaturelimit) = getTemperature();
```

File: scripts/pi-hole/php/header.php [hunk 3]
```diff
@@ -6,156 +6,74 @@
...
+if (!is_numeric($nproc)) {
+    $cpuinfo = file_get_contents('/proc/cpuinfo');
+    preg_match_all('/^processor/m', $cpuinfo, $matches);
+    $nproc = count($matches[0]);
+}
 
-    if($auth) {
-        // For session timer
-        $maxlifetime = ini_get("session.gc_maxlifetime");
+// Get memory usage
```
Comment (yubiuser) [inline scripts/pi-hole/php/header.php:121]:
```suggestion
// Get memory usage, defined in `header_func.php`
```

**Review by yubiuser — COMMENTED** [review]
This is tough to review...

File: style/themes/lcars.css (hunk 1 above)
Comment (rdwebdesign) [reply style/themes/lcars.css:760]: Yes, but only if not logged in and only using this theme (other themes are untouched).

image

I see no reason to add headers with just 5 buttons.

File: scripts/pi-hole/php/header.php (hunk 2 above)
Comment (rdwebdesign) [reply scripts/pi-hole/php/header.php:104]: I changed the code. The functions were NOT moved to a new file.

File: scripts/pi-hole/php/header.php (hunk 3 above)
Comment (rdwebdesign) [reply scripts/pi-hole/php/header.php:121]: I changed the code. The functions were NOT moved to a new file.

File: style/themes/lcars.css (hunk 1 above)
Comment (yubiuser) [reply style/themes/lcars.css:760]: Ok

**Review by yubiuser — APPROVED** [review]

**Pull request merged**